assets/src/components/v2/on_bus/no_data.tsx

```diff
@@ -1,8 +1,26 @@
 import React, { ComponentType } from "react";
-import Placeholder from "../placeholder";
+import { imagePath } from "Util/utils";
 
 const NoData: ComponentType = () => {
-  return <Placeholder color="blue" text="No Data" />;
+  return (
+    <div className="screen-container">
+      <div className="no-data__container">
+        <div className="no-data__header">
+          <div className="no-data__logo-container">
```

Comment:
There are several layers of `div`s that seem unnecessary here:

* The SVG is the only element in the `logo-container`
* The `logo-container` is the only element in the `header`
* The `__text` is the only element in the `main-content`

I think we could collapse these into a less-noisy structure:

* `no-data`
  * `no-data__header`
  * `no-data__content`